web/tests/settings_data.test.js

```diff
@@ -458,43 +466,8 @@ run_test("user_can_access_all_other_users", () => {
     assert.ok(settings_data.user_can_access_all_other_users());
```

Comment:
Test for `can_access_all_users_group` setting does not use the common function because it tests for members and everyone group while other settings do not support everyone group.

We can of-course just set the settings to any groups here for tests irrespective of whether they are allowed in actual code or not, but not sure whether it would be good or not.

And anyways we might allow all the groups later in all the settings, except proably everyone group which will be used for some settings. But we would have enough common options in all the settings to test.

Reply:
I feel like probably we could fix this by making the test read from the data-structure specifying which values are possible for a given setting and skipping those values that are not valid. But probably not worth a lot of effort to do that unless this comes up again.